lib/galaxy_test/selenium/test_custom_builds.py

```diff
@@ -9,7 +9,6 @@ class CustomBuildsTestcase(SharedStateSeleniumTestCase):
     @selenium_test
     def test_build_add(self):
         self._login()
-        self.use_legacy_history()
```

Comment:
Legitimate failure in dev. Custom builds form doesn't load.

<img width="673" alt="Screen Shot 2022-04-14 at 1 26 18 PM" src="https://user-images.githubusercontent.com/216771/163441771-175bb355-7e13-401b-bb7e-f98e9353c07a.png">

Reply:
Looking at this one now.

Reply:
Resolved in #13721.